perspective.el

```diff
@@ -1863,7 +1863,7 @@ restored."
         (frame-count 0)
         (state-complete (read
                          (with-temp-buffer
-                           (insert-file-contents-literally file)
```

Comment:
@zenspider: It looks from `git blame` that you changed `insert-file-contents` to `insert-file-contents-literally` here in December 2019. Do you remember why? I can confirm that it causes the problem @intramurz's PR is trying to fix.

Reply:
Can't say I remember... but looking at the differences in the two, I must have either been having problems with the coding conversion or (my guess) file-name handlers, which `*-literally` overrides with `inhibit-file-name-handlers` ... which just seems backwards to me...

Go ahead and flip it. I'll squock it bites me.